include/wx/uilocale.h

```diff
@@ -40,6 +41,16 @@ class WXDLLIMPEXP_BASE wxLocaleIdent
     // and region separated by dashes.
     static wxLocaleIdent FromTag(const wxString& tag);
 
+    // Add likely subtags to a given locale identifier.
+    static wxLocaleIdent AddLikelySubtags(const wxLocaleIdent& localeIdent);
+
+    // Remove likely subtags from a given locale identifier, favor region.
+    static wxLocaleIdent RemoveLikelySubtags(const wxLocaleIdent& localeIdent, bool favorRegion = true);
```

Comment:
Sorry, but it's always [the same remark](https://www.wxwidgets.org/develop/coding-guidelines/#no_bool_params): please let's avoid functions with non-obvious boolean arguments. I will have *no* idea what `false` means when I see a call `RemoveLikelySubtags(ident, false)`, so this is a bad API